data-raw/11_seas5_empirical_rp_thresholds.R

```diff
@@ -0,0 +1,114 @@
+#' @title: Calculate SEAS5 empirical RP-thresholds
+#' Script calculates empirical return period based threshold from SEAS5 data
+#' by publication/issued month & leadtime for the 5 admins under consideration
+#' Afghanistan drought AA.
+
+#' @note
+#' you will want to `devtools::install_github("OCHA-DAP/cumulus")`
+
+box::use(
+  DBI,
+  RPostgres,
+  cumulus,
+  dplyr[...],
+  purrr[...],
+  lubridate[...],
+  seas5 = ../R/seas5_utils
+)
+
+SEASON_OF_INTEREST <- c(3,4,5)
+AOI_ADM1 <- c(
+  "Takhar",
+  "Badakhshan",
+  "Badghis",
+  "Sar-e-Pul" ,
+  "Faryab"
+)
+
+
+con <- DBI$dbConnect(
+  drv = RPostgres$Postgres(),
+  user = Sys.getenv("DS_AZ_DB_PROD_UID_WRITE"),
+  host = Sys.getenv("DS_AZ_DB_PROD_HOST"),
+  password = Sys.getenv("DS_AZ_DB_PROD_PW_WRITE"),
+  port = 5432,
+  dbname = "postgres"
+)
+
+df_adm1_labels <- tbl(con,"polygon") |>
+  filter(
+    iso3 == "AFG",
+    adm_level == 1
+  ) |>
+  select(
+    iso3, pcode, name
+  ) |>
+  collect()
+
+df_adm1_labels_aoi <- df_adm1_labels |>
+  filter(
+    name %in% aoi_adm1
+  ) |>
+  rename(
+    adm1_name =name
+  )
+
+df_seas5_aoi <- tbl(con, "seas5") |>
+  filter(
+    iso3 == "AFG",
+    adm_level ==1,
+    pcode %in%  df_adm1_labels_aoi$pcode
```

Comment:
I had to actually evaluate `df_adm1_labels_aoi$pcode` with `!!` to get this to work.

Reply:
I have no idea how this makes sense that you would need it and i don't! it should `df_adm1_labels_aoi$pcode` should just be a character string at this point?